pandas/tests/frame/test_sorting.py

```diff
@@ -598,3 +598,71 @@ def test_sort_index_intervalindex(self):
             closed='right')
         result = result.columns.levels[1].categories
         tm.assert_index_equal(result, expected)
+
+    def test_sort_index_na_position_with_categories(self):
```

Comment:
Can you parametrize this test? Believe you can use `values`, `na_position` and `index` as parametrized arguments

Reply:
@WillAyd parametrized a bit. I am pretty new at this. Thanks for all the suggestions!